webapp/src/components/Analysis/UtterancesTable.tsx

```diff
@@ -401,54 +508,95 @@ const UtterancesTable: React.FC<Props> = ({
 
   return (
     <Box className={classes.gridContainer}>
+      <Modal open={modal} onClose={() => showModal(false)}>
+        <Box className={classes.modal}>
+          <Box
+            display="flex"
+            flexDirection="column"
+            alignItems="center"
+            gap={1}
+          >
+            <Typography variant="body1">{modalMessage}</Typography>
+            <Button
+              size="small"
+              sx={{ width: 8 }}
+              variant="contained"
+              onClick={() => {
+                showModal(false);
+              }}
+            >
+              Ok
+            </Button>
+          </Box>
+        </Box>
+      </Modal>
       <div className={classes.gridHeaderActions}>
         <Description
           text="Explore utterances and propose actions. Click on a row to inspect the utterance details."
           link="user-guide/exploration-space/utterance-table/"
         />
-        <Button
-          id="export-button"
-          aria-controls="export-menu"
-          aria-haspopup="true"
-          className={classes.exportButton}
-          onClick={(event) => setAnchorEl(event.currentTarget)}
-          startIcon={<GetApp />}
-          endIcon={<ArrowDropDown />}
-        >
-          Export
-        </Button>
-        <Menu
-          id="export-menu"
-          anchorEl={anchorEl}
-          keepMounted
-          open={Boolean(anchorEl)}
-          onClose={() => setAnchorEl(null)}
-        >
-          <MenuItem
-            onClick={() => {
-              downloadDatasetSplit({
-                jobId,
-                datasetSplitName,
-                ...filters,
-                ...pipeline,
-              });
-              setAnchorEl(null);
-            }}
+        <Box display="flex" alignItems="center" gap={2}>
+          <Button
+            className={classes.exportButton}
+            component="label"
+            startIcon={<UploadIcon />}
+          >
+            Import
+            <input
+              hidden
+              accept=".csv, application/vnd.openxmlformats-officedocument.spreadsheetml.sheet, application/vnd.ms-excel"
```

Comment:
Where did you take that? I don't think we want to accept Excel sheets or anything other than `.csv`. The export comes from Azimuth and we know it has a `.csv` extension.
```suggestion
              accept=".csv"
```